### PR TITLE
Fixing issue #275

### PR DESCRIPTION
**Pull request by megalodonatron**
Added some cute :rabbit: little widgets to portal with more info for people

### CODE REVIEW

File: www/americangut/portal.psp [hunk 1]
```diff
@@ -27,13 +27,32 @@ if message:
...
+                </p>
+                <img src="img/big_participant.png"/>
+            </div>
+        </a>
+    </div>
+    <div class="big_button_div right wfifty">
+        <p class="big_button_text">
+        <ul class="next_steps">
+            <li>Once you've successfully added all of your participants and logged all of your samples, you should send the tubes back to us in the preaddressed envelope.</li>
+            <li>We will then do sequencing which can take up to 6 months, please be patient! We will let you know as soon as your samples have been sequenced and analyzed.</li>
```
Comment (adamrp) [inline www/americangut/portal.psp:51]: The 6 month timeframe sounds reasonable to me, but have we confirmed with someone in the wet lab that this is accurate?

Comment (megalodonatron) [inline www/americangut/portal.psp:51]: Yeah, the quote from Donna, which is in the consent form, is 3-6 months of receipt in the laboratory. I just said up to 6 months to give an overestimate so if it's shorter they'll be happy.

File: www/americangut/portal.psp [hunk 2]
```diff
@@ -27,13 +27,32 @@ if message:
...
-        <div class="big_button_div left wforty">
+    
+    <div class="left wforty">
+        <div class="big_button_div">
             <p class="big_button_text">
-                Add new participants
+                You can add samples to participants using the menu on the left!
             </p>
-            <img src="img/big_participant.png"/>
+            <img src="img/samples.png"/>
```
Comment (adamrp) [inline www/americangut/portal.psp:36]: Can you add this to the repo? or is this a placeholder?

File: www/americangut/portal.psp [hunk 3]
```diff
@@ -27,13 +27,32 @@ if message:
...
+    </div>
+    <div class="big_button_div right wfifty">
+        <p class="big_button_text">
+        <ul class="next_steps">
+            <li>Once you've successfully added all of your participants and logged all of your samples, you should send the tubes back to us in the preaddressed envelope.</li>
+            <li>We will then do sequencing which can take up to 6 months, please be patient! We will let you know as soon as your samples have been sequenced and analyzed.</li>
+            <li>You'll be able to view your results here on this website once they are available.</li>
+        </ul>
+        </p>
+        <img src="img/next_steps.png"/>
```
Comment (adamrp) [inline www/americangut/portal.psp:55]: Same with this -- add to repo?

File: www/americangut/portal.psp (hunk 2 above)
Comment (megalodonatron) [inline www/americangut/portal.psp:36]: added

File: www/americangut/portal.psp (hunk 3 above)
Comment (megalodonatron) [inline www/americangut/portal.psp:55]: added

**Comment by adamrp** [comment]
:+1:   Will wait for other opinions before merge, though

Do we want to provide links to any educational material (e.g., on Illumina sequencing), or will that just confuse people?

**Comment by ElDeveloper** [comment]
:+1:

**Comment by douginator2000** [comment]
+1 Looks good to me. :shipit:

**Pull request merged**